src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py

```diff
@@ -0,0 +1,91 @@
+"""
+Transfer learning example- from our pretrained YoloV5m (on coco) on Pascal VOC. Reaches 68.8 mAP:0.50:0.95.
+"""
+
+import super_gradients
+from super_gradients.training import SgModel, MultiGPUMode
+from super_gradients.training.datasets.dataset_interfaces.dataset_interface import \
+    PascalVOCUnifiedDetectionDataSetInterface
+from super_gradients.training.models.detection_models.yolov5 import YoloV5PostPredictionCallback
+from super_gradients.training.utils.detection_utils import base_detection_collate_fn
+from super_gradients.training.metrics import DetectionMetrics
+from super_gradients.training.utils.detection_utils import Anchors
+
+super_gradients.init_trainer()
+
+distributed = super_gradients.is_distributed()
+
+dataset_params = {"batch_size": 48,
+                  "val_batch_size": 48,
+                  "train_image_size": 512,
+                  "val_image_size": 512,
+                  "val_collate_fn": base_detection_collate_fn,
+                  "train_collate_fn": base_detection_collate_fn,
+                  "train_sample_loading_method": "mosaic",
+                  "val_sample_loading_method": "default",
+                  "dataset_hyper_param": {
+                      "hsv_h": 0.0138,  # IMAGE HSV-Hue AUGMENTATION (fraction)
+                      "hsv_s": 0.664,  # IMAGE HSV-Saturation AUGMENTATION (fraction)
+                      "hsv_v": 0.464,  # IMAGE HSV-Value AUGMENTATION (fraction)
+                      "degrees": 0.373,  # IMAGE ROTATION (+/- deg)
+                      "translate": 0.245,  # IMAGE TRANSLATION (+/- fraction)
+                      "scale": 0.898,  # IMAGE SCALE (+/- gain)
+                      "shear": 0.602,
+                      "mixup": 0.243
+                  }
+                  }
+
+
+model = SgModel("yolov5m_pascal_finetune_augment_fix",
+                multi_gpu=MultiGPUMode.OFF,
+                post_prediction_callback=YoloV5PostPredictionCallback())
+
+dataset_interface = PascalVOCUnifiedDetectionDataSetInterface(dataset_params=dataset_params, cache_labels=True, cache_images=True)
```

Comment:
why caching the labels and images? we usually use false

Reply:
Because we can (pascal is smaller then COCO), and it speeds up training. Its also used in the UL transfer learning example https://github.com/ultralytics/yolov5/issues/1314